demos/smart_framing_demo/cpp_gapi/include/custom_kernels.hpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <opencv2/gapi/gkernel.hpp>
+#include <opencv2/gapi/infer/ie.hpp>
+
+#include <inference_engine.hpp>
+
+#include <utils/slog.hpp>
+
+namespace custom {
+
+struct DetectedObject : public cv::Rect2f
+{
+    unsigned int labelID;
+    std::string label;
+    float confidence;
+};
+
+using GDetections = cv::GArray<DetectedObject>;
+using GLabels = cv::GArray<std::string>;
+
+G_API_OP(GYOLOv4TinyPostProcessingKernel, < GDetections(cv::GMat, cv::GMat, cv::GMat, GLabels, float, float, bool) >, "custom.yolov4_tiny_post_processing") {
```

Comment:
Operation shouldn't be called Kernel, as a Kernel is an implementation to the Operation.